mappings/net/minecraft/enchantment/EnchantmentHelper.mapping

```diff
@@ -1,37 +1,82 @@
 CLASS azu net/minecraft/enchantment/EnchantmentHelper
 	CLASS azu$a Consumer
 		METHOD accept handle (Lazs;I)V
-			ARG 1 level
+			ARG 1 enchantment
+			ARG 2 level
 	METHOD a getHighestApplicableEnchantmentsAtPower (ILawo;Z)Ljava/util/List;
 		ARG 0 power
 		ARG 1 stack
```

Comment:
There's something weird about all the argument mappings below. They don't match the signatures at all. How did this happen?

Reply:
The first argument (level) was incorrectly mapped, and is an Enchantment object type. I believe this error may have initially been my fault to begin with, as there was an error where some parameters would be mapped incorrectly ages ago.

Reply:
I think @JamiesWhiteShirt mean the methods below his comment because they do not match the original methods. For example, onUserDamaged() has only two parameters but in this mapping, there are four.

Reply:
Oh I see. Those mappings are for local variables. Here is what it looks like in Enigma. I did notice that 3 is shown as `integer` instead of `level` though which is odd. @asiekierka Did I mess up?
![image](https://user-images.githubusercontent.com/2250798/50385975-98841000-069b-11e9-82e3-c015023ccd92.png)

Reply:
[Here](https://github.com/FabricMC/yarn/pull/302#issuecomment-447517879) is a comment from @shadowfacts about local variables.